crates/lib/tests/game_ids.rs

```diff
@@ -395,20 +395,20 @@ fn check_definitions_match_name_rules() {
 fn check_node_definitions_match_name_rules() {
     let mut file = fs::OpenOptions::new()
         .read(true)
-        .open("./node-gamedig/games.txt")
+        .open("GAMES_LIST.md")
```

Comment:
Perhaps we should allow this to be set using an environment variable (could still default to this path).

Reply:
Great idea